test/expression/record.stmt

```diff
@@ -0,0 +1,183 @@
+40 columns                              |
+>>> Empty record.
+var record = (   );
+<<<
+var record = ();
+>>> Single-element records don't split after ",".
+var record = (   value  ,  );
+<<<
+var record = (value,);
+>>> Multi-element record.
+var record = (   first  ,  second  ,  third  );
+<<<
+var record = (first, second, third);
+>>> Named record fields.
+var record = (   first  :  1  ,  2 ,  third : 3 );
+<<<
+var record = (first: 1, 2, third: 3);
+>>> Const record.
+var record = const   (  1 ,   2 );
+<<<
+var record = const (1, 2);
+>>> Empty records don't split.
+var longVariableName_______________ = ();
+<<<
+var longVariableName_______________ =
+    ();
+>>> Single-element record.
+var record = (veryLongRecordField________________,);
+<<<
+var record = (
+  veryLongRecordField________________,
+);
+>>> Single-element named record.
+var record = (longFieldName: longRecordFieldValue);
+<<<
+var record = (
+  longFieldName: longRecordFieldValue,
+);
```

Comment:
Yeah, this is the test that should change. Here, we don't need a trailing comma and shouldn't insert one.